### PR TITLE
BMTF dqm  sligtly improved

### PR DESCRIPTION
**Pull request by gkaratha**
Following the recommendation, I make the PR in master branch

### CODE REVIEW

**Comment by gkaratha** [comment]
@thomreis please consider the previous PR (#20150) as a backport. There are no differences between this PR and #20150. Thanks.

**Comment by gkaratha** [comment]
backport of #20185

File: DQM/L1TMonitor/src/L1TStage2BMTF.cc [hunk 1]
```diff
@@ -132,23 +133,25 @@ void L1TStage2BMTF::analyze(const edm::Event & eve, const edm::EventSetup & eveS
...
  
-	  if(itMuon->hwPhi()*0.010902>=0 && itMuon->hwPhi()*0.010902<=30)
+	  /* if(itMuon->hwPhi()*0.010902>=0 && itMuon->hwPhi()*0.010902<=30)
 	    global_phi = itMuon->hwPhi() + itMuon->processor()*30.;
 	  if(itMuon->hwPhi()*0.010902<0)
 	    global_phi = 30-itMuon->hwPhi() + (itMuon->processor()-1)*30.;
 	  if(itMuon->hwPhi()*0.010902>30)
-	    global_phi = itMuon->hwPhi()-30 + (itMuon->processor()+1)*30.;
+	    global_phi = itMuon->hwPhi()-30 + (itMuon->processor()+1)*30.;*/
+	  global_phi= itMuon->hwPhi() + itMuon->processor()*48.-15;
```
Comment (dmitrijus) [inline DQM/L1TMonitor/src/L1TStage2BMTF.cc]: Don't use tabs (never use tabs, disables them from your text editor), it screws up indentation.

File: DQM/L1TMonitor/src/L1TStage2BMTF.cc [hunk 2]
```diff
@@ -132,23 +133,25 @@ void L1TStage2BMTF::analyze(const edm::Event & eve, const edm::EventSetup & eveS
 	  bmtf_hwPt->Fill(itMuon->hwPt());
 	  bmtf_hwQual->Fill(itMuon->hwQual());
 	  bmtf_proc->Fill(itMuon->processor());
-
+          if (fabs(bmtfMuon->getLastBX()-bmtfMuon->getFirstBX())>3){
```
Comment (dmitrijus) [inline DQM/L1TMonitor/src/L1TStage2BMTF.cc]: Please shift the entire block by 2 spaces. It is confusing and unreadable now.

**Comment by gkaratha** [comment]
Dear @dmitrijus ,
I have addressed the comments. Thanks

File: DQM/L1TMonitor/src/L1TStage2BMTF.cc [hunk 3]
```diff
@@ -134,24 +134,24 @@ void L1TStage2BMTF::analyze(const edm::Event & eve, const edm::EventSetup & eveS
 	  bmtf_hwQual->Fill(itMuon->hwQual());
 	  bmtf_proc->Fill(itMuon->processor());
           if (fabs(bmtfMuon->getLastBX()-bmtfMuon->getFirstBX())>3){
```
Comment (thomreis) [inline DQM/L1TMonitor/src/L1TStage2BMTF.cc]: The if and also line 137 should be shifted one space to the left.

File: DQM/L1TMonitor/src/L1TStage2BMTF.cc [hunk 4]
```diff
@@ -134,24 +134,24 @@ void L1TStage2BMTF::analyze(const edm::Event & eve, const edm::EventSetup & eveS
...
-	  /* if(itMuon->hwPhi()*0.010902>=0 && itMuon->hwPhi()*0.010902<=30)
+        /*if(itMuon->hwPhi()*0.010902>=0 && itMuon->hwPhi()*0.010902<=30)
 	    global_phi = itMuon->hwPhi() + itMuon->processor()*30.;
 	  if(itMuon->hwPhi()*0.010902<0)
 	    global_phi = 30-itMuon->hwPhi() + (itMuon->processor()-1)*30.;
 	  if(itMuon->hwPhi()*0.010902>30)
 	    global_phi = itMuon->hwPhi()-30 + (itMuon->processor()+1)*30.;*/
 	  global_phi= itMuon->hwPhi() + itMuon->processor()*48.-15;
-	       if (global_phi<0) global_phi=576+global_phi;
+          if (global_phi<0) global_phi=576+global_phi;
```
Comment (thomreis) [inline DQM/L1TMonitor/src/L1TStage2BMTF.cc]: Shift the if one space to the left please.

**Comment by gkaratha** [comment]
@thomreis thanks for the comments! There were a lot tabs in the other lines. So I replaced every tab with spaces.

File: DQM/L1TMonitor/src/L1TStage2BMTF.cc [hunk 5]
```diff
@@ -125,34 +126,36 @@ void L1TStage2BMTF::analyze(const edm::Event & eve, const edm::EventSetup & eveS
...
-    	  bmtf_hwGlobalPhi->Fill(global_phi);
-	  bmtf_hwEta_hwGlobalPhi->Fill(itMuon->hwEta(), global_phi);
-	}
+        /*if(itMuon->hwPhi()*0.010902>=0 && itMuon->hwPhi()*0.010902<=30)
+            global_phi = itMuon->hwPhi() + itMuon->processor()*30.;
+          if(itMuon->hwPhi()*0.010902<0)
+            global_phi = 30-itMuon->hwPhi() + (itMuon->processor()-1)*30.;
+          if(itMuon->hwPhi()*0.010902>30)
+            global_phi = itMuon->hwPhi()-30 + (itMuon->processor()+1)*30.;*/
+      global_phi= itMuon->hwPhi() + itMuon->processor()*48.-15;
```
Comment (thomreis) [inline DQM/L1TMonitor/src/L1TStage2BMTF.cc]: Here the indenting should be different.

**Comment by thomreis** [comment]
Hi @gkaratha 
thanks. Looks better. You missed two lines though.

**Comment by gkaratha** [comment]
Hi @thomreis 

Many thanks! I fixed it now

**Comment by thomreis** [comment]
@gkaratha thanks,
don't forget to add these changes also in the backport #20150

**Comment by thomreis** [comment]
Can someone launch the tests for this please?

**Comment by dmitrijus** [comment]
+1

The changes are okay, but the code is poorly styled. Also, would be nice is someone applied the code checks.

**Comment by thomreis** [comment]
In order not to touch this PR anymore since it is signed now the patch for the code-checks is applied in a separate PR #20252 .

**Comment by davidlange6** [comment]
+1

**Pull request merged**